src/store.ts

```diff
@@ -90,7 +91,7 @@ export class Store<
     TRootState,
     TOptions extends { modules?: infer T } ? T : undefined
   >
-> implements VuexStore<TRootState> {
```

Comment:
I'm not sure what problem this change fixes. Could you please explain?

Reply:
Nevermind. This change is necessary because TSC can't tell that the TStore#state property is still compatible with the Store#state property.